phalcon/mvc/micro.zep

```diff
@@ -1152,6 +1169,18 @@ class Micro extends Injectable implements \ArrayAccess
 	}
 
 	/**
+	 * Appends a custom 'reponse' handler to be called insted of the default reponse handler
+	 *
+	 * @param callable handler
+	 * @return \Phalcon\Mvc\Micro
+	 */
+	public function response(handler) -> <Micro>
```

Comment:
I did try to keep it the same as "after", "before" and "finish" functions.
But if that is how we should name function, I will of course change it.